src/librustdoc/html/render/context.rs

```diff
@@ -168,18 +168,17 @@ impl<'tcx> Context<'tcx> {
     }
 
     fn render_item(&self, it: &clean::Item, pushname: bool) -> String {
```

Comment:
nit: I would rename `pushname` to `is_module` (and change the uses to `!is_module`), it wasn't clear to me at first why it would be false.